tests/tests.sh

```diff
@@ -101,12 +101,18 @@ test_essentials_telemetry_logging_is_found_on_disk() {
   assertEquals "0" "$?"
 }
 
-# JENKINS-50294 Health checking
+# not used for health-checking anymore, but kept for smoke testing
 test_login_http_200() {
   status_code=$( curl --silent --output /dev/null --write-out "%{http_code}" "http://localhost:$TEST_PORT/login" )
   assertEquals "0" "$?"
   assertEquals "200" "$status_code"
 }
+# JENKINS-50294 Health checking
+test_instance_identity_http_200() {
+  status_code=$( curl --silent --output /dev/null --write-out "%{http_code}" "http://localhost:$TEST_PORT/instance-identity/" )
```

Comment:
Note that the JEP refers in a couple of cases to `/instance-identity`, which is wrong—that would be a redirect to `/instance-identity/`.

Reply:
Fixed, thanks.